server/src/main/java/org/apache/druid/curator/CuratorConfig.java

```diff
@@ -52,6 +52,10 @@
   @JsonProperty("authScheme")
   private String authScheme = "digest";
 
+  @JsonProperty("terminateDruidProcess")
```

Comment:
I meant "terminateDruidProcessOnConnectFail", just replace "quit" part with "terminateDruidProcess"

Reply:
Got it.  Fixed.